labelbox/data/serialization/coco/instance_dataset.py

```diff
@@ -107,6 +107,21 @@ def segmentations_to_common(class_annotations: COCOObjectAnnotation,
                              ])))
     return annotations
 
+def bbox_to_common(class_annotations: COCOObjectAnnotation,
+                            class_name: str) -> List[ObjectAnnotation]:
```

Comment:
Why does this function return a list if it can only ever return a single `ObjectAnnotation`?

Reply:
I wanted to keep the interface created by the other `*_to_common` functions

Reply:
Yeah that consistency is logical and given that we're working in a context of third-party code I agree.

I don't like it at all; The third-party code we're working with is surprisingly poor and I think we're a bit constrained by it. If we had the time I'd prefer to start again using their GraphQL API directly. But we do not! So in this case, I agree with you. Let's keep it as it is.